docs/overrides/main.html

```diff
@@ -2,7 +2,7 @@
 
 {% block announce %}
 👋 Powertools for Python v1 will no longer receive updates or releases after 31/03/2023!
-We encourage you to read our <a href="/upgrade/#end-of-support-v1">upgrade guide on how to migrate to v2</a>.
+We encourage you to read our <a href="https://awslabs.github.io/aws-lambda-powertools-python/latest/upgrade/#end-of-support-v1">upgrade guide on how to migrate to v2</a>.
```

Comment:
Could https://github.com/awslabs/aws-lambda-powertools-python/pull/1893 be a better solution?